tests/sitemaps_tests/test_http.py

```diff
@@ -110,6 +110,38 @@ def test_sitemap_last_modified_mixed(self):
         response = self.client.get('/lastmod-mixed/sitemap.xml')
         self.assertFalse(response.has_header('Last-Modified'))
 
+    def test_sitemaps_lastmod_mixed_ascending_last_modified_missing(self):
+        """
+        Tests that Last-Modified header is omitted when lastmod not found in
```

Comment:
Docs don't explicitly define this behaviour. If we stretch lastmod calculation logic of a `Sitemap` class to this case, all sitemaps should have a lasmod.